config/defaults/dts.json

```diff
@@ -6,7 +6,7 @@
     "template": {
       "embed": {
         "color": "{{ivcolor}}",
-        "title": "{{round iv}}% {{name}} cp:{{cp}} L:{{level}} {{atk}}/{{def}}/{{sta}} {{boostemoji}}",
+        "title": "{{round iv}}% {{name}}{{#if form}}{{#isnt formname 'Normal'}} {{formname}}{{/isnt}}{{/if}}  cp:{{cp}} L:{{level}} {{atk}}/{{def}}/{{sta}} {{boostemoji}}",
```

Comment:
Is the extra whitespace indented? 
Between the `{{/if}}` and `cp:{{cp}` there are 2 whitespaces. (`{{/if}}  cp:{{cp}}`)

Reply:
ty, fixed